### PR TITLE
add editorconfig

### PR DESCRIPTION
**Pull request by OmgImAlexis**
@SickRage/collaborators I'm not sure about the Python rules so this may need to be changed but this should fix the constant space -> tab commits.

http://editorconfig.org/

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
```
Comment (miigotu) [inline .editorconfig:10]: trim_trailing_whitespace = true

Comment (OmgImAlexis) [inline .editorconfig:10]: Done.

**Comment by miigotu** [comment]
Next time pls use a feature branch and not develop/master as your source branch name. It confuses github and it thinks there is some pr with develop in wrong places.

**Comment by OmgImAlexis** [comment]
Guessing that's a new thing? I've always used the dev branch.

**Comment by miigotu** [comment]
For example, one of them is if you go to https://github.com/SickRage/SickRage/tree/develop you'll see #915 next to the branch dropdown.
I know it messes up other places in worse ways, just cant remember where atm.

**Comment by OmgImAlexis** [comment]
Ugh, okay I'll try and use a feature branch from now on.

**Pull request merged**